lib/src/kbasesearchengine/main/SearchMethods.java

```diff
@@ -289,6 +291,18 @@ public SearchObjectsOutput searchObjects(SearchObjectsInput params, String user)
         }
         ret.withTotal((long)hits.total);
         ret.withSearchTime(System.currentTimeMillis() - t1);
+
+        if(user != null) {
+            getLogger().info("user: {}", user );
```

Comment:
Please consolidate the log messages for one query into a single log entry. So that the logs from line 296, 300 and 303 are a single log call.